spinta/core/config.py

```diff
@@ -579,6 +579,7 @@ def configure_rc(
     manifests: List[Union[str, ManifestPath]] = None,
     *,
     mode: Mode = Mode.internal,
+    check_names: bool = False,
```

Comment:
```suggestion
    check_names: Optional[bool] = None,
```

This option can be set via environment variables or other means, so we only should set this, if it is given via command line argument.